### ISSUE TITLE
Capture WaitTime in Server Timings

### ISSUE THREAD

**Issue by dgosbell** [DaxStudio/DaxStudio#1103]
As per the information in Chris Webbs post here https://blog.crossjoin.co.uk/2023/05/02/understanding-waittime-in-power-bi/#comment-130208 the Server Timings trace should capture and display the wait time if it is found in the QueryEnd event

**Comment by marcosqlbi** [comment]
Because it's DAX query-related information, I would plan to store this information by query captured so that it shows up in the "All Queries" window (there should be a column for that) and in the left-area of the server timings pane (which is going to have too many numbers soon).

**Comment by dgosbell** [comment]
Yes, I've got a local branch where I've added the wait time to both All Queries and Server Timings.
